pkg/statistics/hot_peer_cache.go

```diff
@@ -115,6 +116,7 @@ func (f *HotPeerCache) UpdateStat(item *HotPeerStat) {
 		return
 	}
 	f.incMetrics(item.actionType, item.StoreID)
+	f.removeExpiredItems()
```

Comment:
Will it be called every time we receive a region heartbeat?

Reply:
L554-L556